platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationImplTest.java

```diff
@@ -108,10 +108,21 @@ public void testMissingVersionInName() throws Exception {
     @Test(expected = Exception.class)
     public void testUnparsableApplicationName() throws IOException {
         Repository repo = mock(Repository.class);
-        when(repo.getName()).thenReturn("MyName-0.0.0-1.1");
```

Comment:
Not sure why you changed this test case? It looks like it should work with the above change, if I'm reading the [String.split()](https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#split(java.lang.String,%20int)) documentation correctly. The result should be `["MyName", "0.0.0-1.1"]`.